Frontend/src/pages/waiting/components/WaitingList.jsx

```diff
@@ -27,12 +27,26 @@ export default function WaitingList() {
         </Table.Row>
       </Table.Header>
       <Table.Body>
-        {waiting.map((w, i) => (
-          <Table.Row key={w.user_id}>
-            <Table.Cell>{w.user_id}</Table.Cell>
-            <Table.Cell>{i}</Table.Cell>
-          </Table.Row>
-        ))}
+        {waiting ? (
```

Comment:
I'm not sure when this would be null vs empty, but presumably both should be handled the same way?
```suggestion
        {waiting?.length ? (
```